packages/plugin-core/src/test/suite-integ/TogglePreviewLock.test.ts

```diff
@@ -5,81 +5,87 @@ import { TogglePreviewLockCommand } from "../../commands/TogglePreviewLock";
 import { expect } from "../testUtilsv2";
 import { describeSingleWS } from "../testUtilsV3";
+import { PreviewPanel } from "../../views/common/preview/PreviewPanel";
+import sinon from "sinon";
 
 suite("GIVEN TogglePreviewLock", function () {
   let previewPanel: PreviewPanel;
   let cmd: TogglePreviewLockCommand;
 
   beforeEach(() => {
-    previewPanel = PreviewPanelFactory.create(
-      ExtensionProvider.getExtension()
-    ) as PreviewPanel;
+    previewPanel = PreviewPanelFactory.create() as PreviewPanel;
     cmd = new TogglePreviewLockCommand(previewPanel);
   });
 
   afterEach(async () => {
     await VSCodeUtils.closeAllEditors();
   });
 
-  describeSingleWS("WHEN locking preview from the command bar", {}, () => {
-    beforeEach(() => {
-      previewPanel.unlock(); // reset
-    });
-    describe("AND preview is hidden", () => {
-      test("THEN preview should be NOT locked", async () => {
-        await cmd.run();
-        expect(previewPanel.isLocked()).toBeFalsy();
+  describeSingleWS(
+    "WHEN locking preview from the command bar",
+    { timeout: 5e3 },
+    () => {
+      beforeEach(() => {
+        previewPanel.unlock(); // reset
       });
-    });
-
-    describe("AND preview just opened", () => {
-      let note1: NoteProps;
-      let note2: NoteProps;
-      beforeEach(async () => {
-        const { engine, wsRoot, vaults } = ExtensionProvider.getDWorkspace();
-        note1 = await NoteTestUtilsV4.createNoteWithEngine({
-          engine,
-          wsRoot,
-          vault: vaults[0],
-          fname: "preview-test",
+      describe("AND preview is hidden", () => {
+        test("THEN preview should be NOT locked", async () => {
+          await cmd.run();
+          expect(previewPanel.isLocked()).toBeFalsy();
         });
-        await ExtensionProvider.getWSUtils().openNote(note1);
-        await previewPanel.show(note1);
-      });
-      test("THEN preview should be locked and pristine", async () => {
-        await cmd.run();
-        expect(await previewPanel.isLockedAndDirty()).toBeFalsy();
       });
-      describe("AND has been locked", () => {
+
+      describe("AND preview just opened", () => {
+        let note1: NoteProps;
+        let note2: NoteProps;
         beforeEach(async () => {
-          await cmd.run();
+          const { engine, wsRoot, vaults } = ExtensionProvider.getDWorkspace();
+          note1 = await NoteTestUtilsV4.createNoteWithEngine({
+            engine,
+            wsRoot,
+            vault: vaults[0],
+            fname: "preview-test",
+          });
+          await ExtensionProvider.getWSUtils().openNote(note1);
+          await previewPanel.show(note1);
         });
-        test("THEN preview should toggle to unlocked", async () => {
+        test.only("THEN preview should be locked and pristine", async () => {
+          const wsUtils =
+            previewPanel._DO_NOT_USE_EXPOSED_FOR_TESTING_wsUtilsWeb();
```

Comment:
Can you add a comment here quickly detailing the issue between having 2 different wsRoot's being used by `ExtensionProvider.getDWorkspace()` versus `container.resolve()`